src/styles/main.scss

```diff
@@ -1,7 +1,16 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap");
 
-body {
-  background: $c-gray;
-}
+@import "./variables";
+@import "./mixins";
+
+@import "blocks/page";
+@import "./blocks/grid.scss";
+@import "blocks/container";
+
+@import "./blocks/recommended.scss";
+@import "./blocks/products.scss";
+@import "./blocks/button.scss";
+@import "./blocks/about.scss";
+@import "./blocks/contacts.scss";
+@import "./blocks/form-field.scss";
```

Comment:
The file import should be consistent, remove the .scss extension from the import statement.